Content/CBBTemplate/Directory.Build.props

```
@@ -0,0 +1,17 @@
+<Project>
+  <Import Project="build\Version.props" />
+  <PropertyGroup>
+    <LangVersion>latest</LangVersion>
+    <PackageOutputPath>$(MSBuildThisFileDirectory)bin\$(Configuration)</PackageOutputPath>
+    <Company>dotnet 职业技术学院 (dotnet campus)</Company>
+ 
+    <!-- 替换下面信息 -->
+    <Authors>otnet campus</Authors>
```

```suggestion
    <Authors>dotnet-campus</Authors>
```

This name is used by the nuget.org. So it should be the same with the id.